Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -158,16 +157,20 @@ public function storeOutput()
             $this->logger->debug("Default bucket " . $uploadTablesOptions["bucket"]);
         }
 
-        $writer->uploadTables($this->dataDirectory . "/out/tables", $uploadTablesOptions, $systemMetadata);
         try {
-            $writer->uploadFiles($this->dataDirectory . "/out/files", ["mapping" => $outputFilesConfig]);
-        } catch (ManifestMismatchException $e) {
-            $this->logger->warning($e->getMessage());
-        }
+            $writer->uploadTables($this->dataDirectory . "/out/tables", $uploadTablesOptions, $systemMetadata);
+            try {
```

Comment:
v testech to něco rozbije? pokud ne, můžeme dohledat warningy, jestli se někde dějou, a sjednotit to